UnityProject/Assets/Scripts/Systems/Construction/ComputerCircuitboard.cs

```diff
@@ -6,15 +7,35 @@ namespace Items.Construction
 	/// Allows an object to function as a circuitboard for a computer, being placed into a computer frame and
 	/// causing a particular computer to be spawned on completion.
 	/// </summary>
-	public class ComputerCircuitboard : MonoBehaviour
+	public class ComputerCircuitboard : MonoBehaviour, ICheckedInteractable<HandApply>
 	{
 		[Tooltip("Computer which should be spawned when this circuitboard's frame is constructed.")]
 		[SerializeField]
 		private GameObject computerToSpawn = null;
 
+		[SerializeField] private ItemTrait screwdriver;
+		[SerializeField] private List<GameObject> customTypes = new List<GameObject>();
```

Comment:
so nothing uses currently? , could you hook at the script on the computer frame to use this if it set

Reply:
..it already does tho?

Reply:
computerToSpawn  isn't changed?

Reply:
![image](https://user-images.githubusercontent.com/34368774/168452612-839ad3ba-3a28-4fb3-abe0-d21501ad901e.png)

you sure? :)

Reply:
Make sure to set it via awake then , To So it So it picks up from customTypes with the current currentIndex

Reply:
Done 👍